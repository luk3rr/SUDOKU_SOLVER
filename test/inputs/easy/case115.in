020015900 001070026 007030058 000007010 002400587 000001000 830050001 290000005 000003060 
