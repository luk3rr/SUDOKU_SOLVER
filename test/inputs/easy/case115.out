Solving the following grid:
0 2 0 | 0 1 5 | 9 0 0 
0 0 1 | 0 7 0 | 0 2 6 
0 0 7 | 0 3 0 | 0 5 8 
------+-------+------
0 0 0 | 0 0 7 | 0 1 0 
0 0 2 | 4 0 0 | 5 8 7 
0 0 0 | 0 0 1 | 0 0 0 
------+-------+------
8 3 0 | 0 5 0 | 0 0 1 
2 9 0 | 0 0 0 | 0 0 5 
0 0 0 | 0 0 3 | 0 6 0 


Solution found :')

6 2 3 | 8 1 5 | 9 7 4 
5 8 1 | 9 7 4 | 3 2 6 
9 4 7 | 6 3 2 | 1 5 8 
------+-------+------
4 5 9 | 3 8 7 | 6 1 2 
3 1 2 | 4 9 6 | 5 8 7 
7 6 8 | 5 2 1 | 4 9 3 
------+-------+------
8 3 6 | 7 5 9 | 2 4 1 
2 9 4 | 1 6 8 | 7 3 5 
1 7 5 | 2 4 3 | 8 6 9 

Algorithm: BFS
Total time: 31 ms
Total expanded states: 3637
